alerts/state.libsonnet

```diff
@@ -60,7 +62,9 @@
               severity: 'warning',
             },
             annotations: {
-              message: 'There are {{ $value }} different versions of Ceph Mon components running.',
+              message: 'There are multiple versions of Ceph Mon running.',
```

Comment:
We try avoiding name `ceph` in alert messages. The message could be generic enough in nature saying `Different versions of storage services running`

Reply:
Done